tst/EnergyPlus/unit/RoomAirflowNetwork.unit.cc

```diff
@@ -249,8 +249,8 @@ TEST_F( RoomAirflowNetworkTest, RAFNTest )
 
 	Surface( 1 ).HeatTransferAlgorithm = HeatTransferModel_EMPD;
 	Surface( 2 ).HeatTransferAlgorithm = HeatTransferModel_EMPD;
-	MoistEMPDNew( 1 ) = 0.0011;
-	MoistEMPDNew( 2 ) = 0.0012;
```

Comment:
@nmerket Here is a section of code in the CalcSurfaceMoistureSums function of the RoomAirflowNetwork module

			if ( Surface( SurfNum ).HeatTransferAlgorithm == HeatTransferModel_EMPD ) {

				UpdateMoistureBalanceEMPD( SurfNum );
				RhoVaporSurfIn( SurfNum ) = MoistEMPDNew( SurfNum );

				SumHmAW = SumHmAW + HMassConvInFD( SurfNum )*Surface( SurfNum ).Area* ( RhoVaporSurfIn( SurfNum ) - RhoVaporAirIn( SurfNum ) );
				SumHmARa = SumHmARa + HMassConvInFD( SurfNum )*Surface( SurfNum ).Area* PsyRhoAirFnPbTdbW( OutBaroPress, TempSurfInTmp( SurfNum ),
					PsyWFnTdbRhPb( TempSurfInTmp( SurfNum ), PsyRhFnTdbRhovLBnd0C( TempSurfInTmp( SurfNum ), RhoVaporAirIn( SurfNum ) ), OutBaroPress ) );
				SumHmARaW = SumHmARaW + HMassConvInFD( SurfNum )*Surface( SurfNum ).Area*RhoVaporSurfIn( SurfNum );
			}